templates/base.html

```diff
@@ -95,6 +98,18 @@
                   <a class="nav-link" href="/{{href}}">{{id}}</a>
               </li>
               {% endfor %}
+              {% if organization_bar %}
+              <li class="nav-item dropdown {% if  active_page == 'Organization' %}active{% endif %}">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                  Organization
+                </a>
+                <div class="dropdown-menu dropdown-menu-lg-right" aria-labelledby="navbarDropdown">
```

Comment:
aria-labelledby should be updated to match organization drop down id

Reply:
Agreed. Will submit in fix momentarily.